lib/Common/Memory/RecyclerPointers.h

```diff
@@ -220,6 +235,9 @@ class NoWriteBarrierField
     // Setters
     NoWriteBarrierField& operator=(T const& value)
     {
+#if defined(RECYCLER_WRITE_BARRIER)
+        RecyclerWriteBarrierManager::VerifyIsBarrierAddress(this);
```

Comment:
Should this be VerifyIs"Not"? Should this be under #if defined(RECYCLER_WRITE_BARRIER) && ENABLE_DEBUG_CONFIG_OPTIONS, && commandline options?